libraries/rush-lib/src/cli/actions/RemoveAction.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as os from 'os';
+import { CommandLineFlagParameter, CommandLineStringListParameter } from '@rushstack/ts-command-line';
+
+import { RushConfigurationProject } from '../../api/RushConfigurationProject';
+import { BaseRushAction } from './BaseRushAction';
+import { RushCommandLineParser } from '../RushCommandLineParser';
+
+import type * as PackageJsonUpdaterType from '../../logic/PackageJsonUpdater';
+
+export class RemoveAction extends BaseRushAction {
```

Comment:
This is pretty similar to `AddAction`. Can these be DRYed up with a common base?

Reply:
done